test_fakeredis.py

```diff
@@ -3078,6 +3078,36 @@ def _listen(pubsub, q):
         received = q.get()
         self.assertEqual(received['data'], msg)
 
+    @attr('slow')
+    def test_pubsub_run_in_thread(self):
+        q = Queue()
+
+        def _queue_msg(msg, q):
+            q.put(msg)
+
+        pubsub = self.redis.pubsub()
+        pubsub.subscribe(**{"channel": lambda e: _queue_msg(e, q)})
```

Comment:
Instead of the lambda and `_queue_msg`, you could just pass `q.put` as the callback.

Reply:
I'm not sure how to pass the event object to the callback if I just use q.put as the callback in this context. I could remove the`_queue_msg` function and simply make it `lambda e: q.put(e)`, however.

Reply:
fixed